doc/index.rst

```diff
@@ -101,18 +107,20 @@ Here are some operations in your client repositories.
 To start using CI config stored in this repo
 --------------------------------------------------
 
-With the following few short steps, you can start in your client repository using CI confiurations stored in here (`industrial_ci` repository).
+With the following few short steps, you can start in your client repository using CI configurations stored in here (`industrial_ci` repository).
 
-1. Don't forget to activate Travis CI on your github repository (you may do so on https://travis-ci.org/profile/YOUR_GITHUB_ORGANIZATION or https://travis-ci.org/profile/YOUR_GITHUB_USER).
+1. Don't forget to activate CI for your repository.
 
-2. In `.travis.yml` file in your client repo, add in `before_config` section a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .ci_config`, like below:
+- For Travis CI and GitHub, you may do so on https://travis-ci.org/profile/YOUR_GITHUB_ORGANIZATION or https://travis-ci.org/profile/YOUR_GITHUB_USER (replace capital with your value).
 
+2. In `CI config <#terminology>`_ file in your client repo, add in `before_config` section a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .ci_config`, like below:
 ::
 
   before_config:
     - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
   script:
-    - .ci_config/travis.sh
```

Comment:
This is confusing. Don't mix non-travis scripts with travis(-like) config.

I would suggest to don't touch this section for now and to improve it in another PR.
I guess we should have an extended quickstart for all supported CI systems.